src/contrasts.jl

```diff
@@ -22,7 +22,7 @@ instantiation optionally includes the levels to generate columns for and the bas
 level. If not specified these will be taken from the data when a `ContrastsMatrix` is
 generated (during `ModelFrame` construction).
 
-# Constructors
+**Constructors**
```

Comment:
It seems more natural IMO to separate sections like this by using markdown titles (doesn't have to be h1) rather than bold. Typically it's `#` for main title, `##` for sections, and `###` for subsections.